src/routes/docs/products/messaging/messages/+page.markdoc

```diff
@@ -0,0 +1,246 @@
+---
+* `color`
+* optional
+* Specifies a color tint for the notification icon, used only for Android devices. This can be used to align with brand colors.
+---
+* `tag`
+* optional
+* Can be used to replace an existing notification with the same tag, used only for Android devices. Useful for updating or canceling notifications.
+---
+* `badge`
+* optional
+* Sets the number to display next to the app's icon, indicating the number of notifications or updates. Setting to 0 removes any existing badge. This is for Apple devices only.
+---
+* `status`
+* optional
+* The status of the push notification, which can be `draft`, `scheduled`, or `processing` (default). `Processing` means the notification is sent immediately.
+---
+* `scheduledAt`
+* optional
+* An ISO date time string specifying when the push notification should be sent. This parameter must be set if the status is `scheduled` and must represent a future time.
+{% /table %}
+
+{% /tabsitem %}
+{% tabsitem #emails title="Emails" %}
+
+{% table %}
+* Parameter {% width=100 %}
+* Required  {% width=100 %}
+* Description
+---
+* `subject`
+* required
+* The subject line of the email. This is what recipients see first in their inbox.
+---
+* `content`
+* required
+* The main content of the email. This can be plain text or HTML, depending on the `html` flag.
+---
+* `cc` 
+* optional
+* An array of target IDs to be included in the carbon copy (CC) field. These recipients can see each other's email addresses.
+---
+* `bcc` 
+* optional 
+* An array of target IDs to be included in the blind carbon copy (BCC) field. These recipients cannot see each other's email addresses.
+---
+* `html` 
+* optional
+* A boolean indicating whether the `content` is in HTML format. This allows for rich text, links, and styling in the email content.
+---
+* `status` 
+* optional
+* The status of the email, which can be draft, scheduled, processing (default). Processing means the email is sent immediately.
+---
+* `scheduledAt`
+* optional
+* An ISO date time string specifying when the email should be sent. This parameter must be set if the status is `scheduled` and must represent a future time.
+{% /table %}
+
+{% /tabsitem %}
+{% tabsitem #sms title="SMS" %}
+{% table %}
+* Parameter {% width=100 %}
+* Required  {% width=100 %}
+* Description
+---
+* `content`
+* required
+* The main content of the SMS. This should be concise and clear, as SMS messages have character limits.
+---
+* `status`
+* optional
+* The status of the SMS message, which can be `draft`, `scheduled`, or `processing` (default). `Processing` means the SMS is sent immediately.
+---
+* `scheduledAt`
+* optional
+* An ISO date time string specifying when the SMS should be sent. This parameter must be set if the status is `scheduled` and must represent a future time.
+{% /table %}
+{% /tabsitem %}
+{% /tabs %}
+
+# Sending a message {% #create-a-message %}
+You can create a message with the `METHOD` method using a Server SDK.
+{% multicode %}
+```js
```

Comment:
@abnegate @stnguyen90 Simple code example of just the three methods.

We will cover full flow in detail in their journies.

Reply:
You should be able to determine the methods from the [API](https://github.com/appwrite/appwrite/blob/49a753509b17ee4209999e08678e8cad71a20bee/app/controllers/api/messaging.php#L2275-L2306). Do you need additional detail?